### ISSUE TITLE
[Star Wars Saga Edition] Several fixes of varying types

### ISSUE THREAD

**Issue by LilLiteralist** [Roll20/roll20-character-sheets#10617]
I'm not sure if these should be submitted as separate issues, but some of these things have been driving me nuts for awhile. Some are more recent.

EDIT: At this point, I have fixed all of the original issues. There are still the issues in the comments, though.

1. ~~With the addition of the Second Wind stuff, the spacing is a bit cramped with the radio buttons under Condition. This is the default spacing when the sheet is opened.~~
![image](https://user-images.githubusercontent.com/104447108/170686328-6dd52755-f956-40f4-9593-78dd8b8ac9ef.png)
2. ~~Right now, `attr_damage_max` adds half the character level (which is in `@{level|max}` and factors in nonheroic levels). This should be changed to half the _heroic_ level (which is currently found in `@{level}`), rounded down.~~
3. ~~The checkbox for "Dex score as decimal" should actually use "Initiative modifier as decimal," since that is the method for breaking ties in this system.~~
4. ~~Since droids do not have a Con score, it would be good if we can have that represented on the sheet as an entry of "-". The modifer would need to be set to 0 in that case. This could be worked around easily by just putting in a score of 10, but the new Use Second Wind button does not take into account that droids will never use their Constitution score.~~
5. ~~There's a checkbox for double carrying capacity, but this doesn't come into play very often. A 2x modifier can come from a particular 5th degree droid talent (unused in any statblock and unavailable to anyone but the right type of droid) or from low gravity. High gravity has a carrying capacity of 0.5x normal, and zero gravity has a carrying capacity of 10x normal, but changes in gravity are not common in this system. A far more used modifier is 1.5x from Extra Legs, which is still limited to droids, but at least available to every type of droid and is actually used in official statblocks. Just changing this to 1.5x modifier would be better than it is currently. I don't know if taking gravity or that one droid talent into account is worth putting in other options.~~
6. ~~Once a PC hits level 10, Armored Defense (without Improved Armored Defense) will only calculate correctly if the armor bonus to Ref is 0 or 1. Once the PC hits level 20, it will also calculate correctly if the armor bonus to Ref is 0, 1, or 2. And if you put in level 30, it's good for an armor bonus of 0-3. 
![image](https://user-images.githubusercontent.com/104447108/170689115-dcb651bb-8a21-47f8-9629-3559587328c1.png)
This is where the issue is, though I have no idea why it only seems to be using the first digit of the level's value.~~

**Comment by LilLiteralist** [comment]
Oh, and fix "Encumberance" to "Encumbrance"

**Comment by LilLiteralist** [comment]
I've managed to fix most of these issues myself, though I've found a host of others as well.

1. Make the print function not look like a hot mess.
2. Add support for translation in the sheet. (Lots of data-i18n)
3. Make sure mouseover text has variable names that can be called in macros
4. Status button below Perception and Initiative which sends combat-relevant statistics to chat.
5. Vehicle skills need fixing, sometimes using the Pilot modifier if trained, and sometimes using the higher of Pilot and the skill.
6. Relocate the [hide/show] toggle for attacks, so that it isn't behind the trash can when the repeating sections are being modified.
7. Add a spot on the NPC sheet for armor.
8. Ability to select default skill crews in vehicle sheet
9. Remove the armor's Helmet Package bonus to Perception if the character is not proficient.

**Comment by nmbradley** [comment]
@LilLiteralist are these issues still outstanding, following the work on the sheet?

**Comment by LilLiteralist** [comment]
@nmbradley Most of it has been resolved, although I could certainly add more things that this sheet would benefit from. Is there a better place to keep track of standing issues with the sheet?

**Comment by polygoncount** [comment]
@LilLiteralist Seems like the Initiative and Mechanics skill mods on the Vehicle Sheet are strangely entwined, while the Initiative *total* modifier stays blank.

**Comment by LilLiteralist** [comment]
Thanks for catching that. It's been 2 months since that change. I'm surprised no one mentioned it before you.

**Comment by polygoncount** [comment]
Was also considering a request for Knowledge (Tactics) to be added to the Commander's list of available skills

**Comment by polygoncount** [comment]
I should also mention, it appears the "Other Modifiers" prompt for skills AND attacks setting for Vehicles isn't working when selected.

**Comment by polygoncount** [comment]
It appears that the Experience Points section has disappeared from the character sheet.

**Comment by LilLiteralist** [comment]
Yes, I apologize for this. A pull request has been submitted. The default state will be showing XP, of course. Even if I suspect that using no XP is more common.
